Cargo.toml

```diff
@@ -211,15 +211,17 @@ lto = "thin"
 # can wire up the 'abrt' signal handler so tempfiles will be removed in case of panics.
 panic = "unwind"
 incremental = false
-build-override = { opt-level = 0 }
```

Comment:
I'm not sure I understand the reason for removing this based on the information in the commit message, though I can guess a possible good reason. This was added in add2e3e (#2) and only reduces the optimization level for `build` dependencies, i.e., those that are used for building. So the resulting executables' level of optimization is unaffected by this. But a note in [the documentation](https://doc.rust-lang.org/cargo/reference/profiles.html#overrides) mentions:

> When a dependency is both a normal dependency and a build dependency, Cargo will try to only build it once when `--target` is not specified. When using `build-override`, the dependency may need to be built twice, once as a normal dependency and once with the overridden build settings. This may increase initial build times.

Is this the reason for removing it?

Reply:
I completely misunderstood `build-override` and was thinking it's related to an opt-level override for anything that is built, not just build scripts.

Avoiding to possibly build dependencies twice It's probably a good reason to remove it, especially given that there are no build dependencies with a build-script that just uses the standard library.